examples/filesystem/main.c

```diff
@@ -153,7 +169,7 @@ static int _mount(int argc, char **argv)
 {
     (void)argc;
     (void)argv;
-#if defined(MTD_0) && (defined(MODULE_SPIFFS) || defined(MODULE_LITTLEFS) || defined(MODULE_FATFS_VFS))
+#if defined(MTD_0) && (defined(MODULE_SPIFFS) || defined(MODULE_LITTLEFS) || defined(MODULE_LITTLEFS2) || defined(MODULE_FATFS_VFS))
```

Comment:
This should probably be refactored to some easy define on the top of the file, seeing how it is reused here. Out of scope here though.

Reply:
Yea we should move those outside the filesystem example while we're at it.